src/js/utils/utils.js

```diff
@@ -96,30 +136,50 @@ function isBrowserCompatable () {
     return chromeVersion >= MINIMUM_CHROME_VERSION;
   }
   catch (error) {
+    logger.error('Unable to detect Chrome version');
     logger.error(error);
 
     return false;
   }
 }
 
-function mediaSourceExtensionsCheck () {
-  // For the MAC
-  window.MediaSource = window.MediaSource || window.WebKitMediaSource;
-
-  if (!window.MediaSource) {
-    throw new Error('Media Source Extensions not supported in your browser: Claris Live Streaming will not work!');
-  }
-}
-
+/**
+ * Check to see if the passed mimeType is supported by CLSP Player.
+ *
+ * @param {String} mimeType
+ *   Will check to see if this mimeType is supported
+ *
+ * @returns {Boolean}
+ *   `true` if the mimeType is supported by CLSP Player
+ *   `false` if the mimeType is not supported by CLSP Player
+ */
 function isSupportedMimeType (mimeType) {
   return mimeType === SUPPORTED_MIME_TYPE;
 }
 
-function _getWindowStateNames () {
+/**
+ * @typedef {Object} PageVisibilityApiPropertyNames
+ * @property {String} hiddenStateName
+ *   The property name used for the `document.hidden` property
+ * @property {String} visibilityChangeEventName
+ *   The property name used for the `document.visibilityChange` event name
+ */
+
```

Comment:
@brainthinks is this comment broken up on purpose?

Reply:
that's the way the typedef was demo'd in jsdoc docs:

https://jsdoc.app/tags-typedef.html